.github/workflows/test-demo.yaml

```diff
@@ -33,10 +33,15 @@ jobs:
         run: nix build .#overview
 
       - name: Run and test VM
+        env:
+          DISTRO: ${{ matrix.distro }}
+          NIX_CONFIG: |
+            extra-substituters = https://ngi.cachix.org/
+            extra-trusted-public-keys = ngi.cachix.org-1:n+CAL72ROC3qQuLxIHpV+Tw5t42WhXmMhprAGkRSrOw=
         run: >
           docker run
           --privileged
           --volume ./result/project/Cryptpad/default.nix:/default.nix
           --volume "$(pwd)/.github/workflows/test-demo.sh:/test-demo.sh"
           ${{ matrix.distro }}
```

Comment:
```suggestion
          "$DISTRO"
```

Reply:
Does that make a difference though?

Reply:
Not really, but we already have the the env var, so might as well use it.